HW02_B_Mimic_starter.Rmd

```diff
@@ -52,7 +54,12 @@ data("iris")
 
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
-
+iris$Species2 <-factor(iris$Species, levels=c("versicolor", "setosa", "virginica"))
+ggplot(iris, aes(Sepal.Length, Petal.Length))+
+  geom_point(aes(color=Species, shape=Species))+
+  facet_wrap(~Species2, scales = "free_y")+

```

Comment:
Thanks, this line has been really helpful, since I did not know that the axes of faceted figures can be synced in only one dimension.

Reply:
By the way, it might be a good idea to keep it coherent in how spaces are added, e.g., comparing to line 60, I would recommend "=" to be wrapped with spaces also in line 59.